CONTRIBUTING.md

```diff
@@ -35,3 +35,41 @@ Examples:
 ./run-tests.sh 5.4 ./link
 ```
 
+# Contributor ladder
+
+If you'd like to contribute to the library more regularly, one of the
+[maintainers][ebpf-lib-maintainers] can add you to the appropriate team or mark
+you as a code owner.
+Just create an issue in the repository.
```

Comment:
Nit: can be reflowed.

I'd maybe add #ebpf-go-dev as another avenue for getting ownership.

Reply:
I prefer issues since it leaves an "audit trail" of the decision, which we lose with the slack channel. OTOH it might increase the barrier for people if we reject their request publicly? I could also get rid of this and say "contact the maintainers"